### PR TITLE
[CSL-2139] ekg server: allow 0.0.0.0 as listening address

### PR DESCRIPTION
**Pull request by ksaric**
https://iohk.myjetbrains.com/youtrack/issue/CSL-2139

### CODE REVIEW

**Comment by ksaric** [comment]
@domenkozar I don't have write access to merge this one.

File: scripts/launch/connect-to-cluster/default.nix [hunk 1]
```diff
@@ -84,6 +84,6 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
     ${ ifWallet "--wallet-db-path '${stateDir}/wallet-db'"}        \
     --keyfile ${stateDir}/secret.key                               \
         ${ ifWallet "--wallet-address ${walletListen}"}    \
-        --ekg-server 127.0.0.1:8080 --metrics +RTS -T -RTS             
+        --ekg-server 0.0.0.0:8080 --metrics +RTS -T -RTS
```
Comment (domenkozar) [inline scripts/launch/connect-to-cluster/default.nix]: Hmm, actually we should default to 127.0.0.1 and make this overridable (which would be set to 0.0.0.0 for docker, same as for walletListen).

Comment (ksaric) [reply scripts/launch/connect-to-cluster/default.nix]: Is `ekgServerListen` good?

Comment (domenkozar) [reply scripts/launch/connect-to-cluster/default.nix]: Yup!

**Comment by domenkozar** [comment]
@ksaric I'll pick this up as I have a few other fixes

**Comment by domenkozar** [comment]
@ksaric I hope you don't mind stealing your commit without proper attribution, feel free to own the first one.

**Pull request closed without merging**